app/src/main/java/org/p2p/wallet/auth/ui/pin/signin/SignInPinFragment.kt

```diff
@@ -40,9 +44,18 @@ class SignInPinFragment :
             requireActivity().finish()
         }
         with(binding) {
+            with(toolbar) {
+                setOnMenuItemClickListener {
+                    if (it.itemId == org.p2p.wallet.R.id.helpItem) {
+                        org.p2p.wallet.intercom.IntercomService.showMessenger()
```

Comment:
you can remove package definition here and line above

Reply:
strange thing = ( will remove this mess